addon/components/frost-chart-svg-plot-x-grid.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Component definition for the frost-chart-svg-plot-x-grid component
+ */
+import Ember from 'ember'
+const {get} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {isDomainValid} from 'ember-frost-chart/utils/validation'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+
+import layout from '../templates/components/frost-chart-svg-plot-x-grid'
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  layout,
+  tagName: 'g',
+
+  // == PropTypes =============================================================
+
+  /**
+   * Properties for this component. Options are expected to be (potentially)
+   * passed in to the component. State properties are *not* expected to be
+   * passed in/overwritten.
+   */
```

Comment:
Again, really minor - but I strip the unused boilerplate (comments, actions block, etc.) out of new component files so that you don't have to browse past them when looking for actual code.